lib/plugins/blocks.js

```diff
@@ -454,6 +463,54 @@ function inject (bot, { version, storageBuilder }) {
     })
   }
 
+  /**
+   * Places a sign and writes to it
+   * @throws {Error} Throws error when no sign is equipped or the position to place the sign is is already occupied
+   * @param {import('vec3').Vec3} pos
+   * @param {string} text
+   * @param { { sneak?: boolean, writeDelay?: number } } options
+   * @returns {Promise<void>}
+   */
+  bot.placeSign = async (pos, text, options = {}) => {
+    if (!(pos instanceof Vec3)) throw new Error('Invalid argument for pos. Must be a instance of Vec3 got ' + typeof pos)
```

Comment:
I think more checks are never wrong if they do not interfere with performance or other things. I do not expect people to run placeSign hundreds off times a second so performance should not be an issue. But I can also remove that check again.

Reply:
Yes, I understand that. I meant from a consistency standpoint to have clear code. If it's a common problem it may make sense, otherwise it sounds like noise to me if some have these checks and some don't

Reply:
I feel like a lot off problems for people that are new to mineflayer come from sometimes really obscure error messages. Catching the really obvious stuff should probably be implemented for every api facing function where it dose not hinder performance.

Reply:
Perhaps a discussion for another time, there are type checkers for this purpose